cmd/ingress/ingress.go

```diff
@@ -45,6 +46,15 @@ var approvedTraders = []string{
 	"513167dd7C2B1110e4Ec212E79c430eE72efeCf2",
 }
 
+func init() {
+	sentryDSN := os.Getenv("SENTRY_DSN")
+	if sentryDSN == "" {
+		log.Println("Could not find SENTRY_DSN env variable. Not using Sentry.")
```

Comment:
This should be a `panic` or `log.Fatalln` because we do not want the Ingress to start without this configuration.